Automattic/jetpack
# Instant Search: display price information on product results

#### Changes proposed in this Pull Request:

* Request additional fields for Woo price when viewing product results.
* Display price and sale price when viewing product results.

The PR does not aim to implement final styling for product prices. We will also have to undertake further testing later with different types of product (subscriptions, free products, and so on).

<img width="693" alt="Screen Shot 2021-02-10 at 16 01 16" src="https://user-images.githubusercontent.com/17325/107459415-0745bf80-6bbb-11eb-8ccd-ec6bcd4916db.png">

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

On an Instant Search-enabled site with WooCommerce installed, search for a product and add`result_format=product` to the query string. Check that the product price is displayed.

<img width="250" alt="Screen Shot 2021-01-08 at 16 03 32" src="https://user-images.githubusercontent.com/17325/103969595-1d7eeb00-51cb-11eb-93ed-0bcf85b8768b.png">

On a product with a sale price, ensure that the regular price is displayed (struck through) along with the sale price.

#### Proposed changelog entry for your changes:


Not required yet. The product result format has not yet been released to end users.
